bin/deploy

```diff
@@ -20,6 +20,7 @@ esac
 
 # Stop RabbitMQ consumers
 bin/console rabbitmq-supervisor:control stop --wait-for-supervisord
+bin/console messenger:stop-workers
```

Comment:
This is actually wrong, this will stop the workers, but then supervisor will restart them right away, but at least the workers will be refreshed if there are code changes.

Do we even have to stop the workers ever at all during deployment? Discuss...

Reply:
This is what they say on the Symfony messenger documentation

`**Restart Workers on Deploy**

Each time you deploy, you’ll need to restart all your worker processes so that they see the newly deployed code. To do this, run messenger:stop-workers on deploy. This will signal to each worker that it should finish the message it’s currently handling and shut down gracefully. Then, Supervisor will create new worker processes. The command uses the app cache internally - so make sure this is configured to use an adapter you like.`